flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/table/FileSystemOutputFormatTest.java

```diff
@@ -147,39 +145,40 @@ public void testStaticPartition() throws Exception {
             testHarness.processElement(new StreamRecord<>(Row.of("a2", 2), 1L));
             testHarness.processElement(new StreamRecord<>(Row.of("a2", 2), 1L));
             testHarness.processElement(new StreamRecord<>(Row.of("a3", 3), 1L));
-            assertEquals(1, getFileContentByPath(tmpFile).size());
+            assertThat(getFileContentByPath(tmpPath)).hasSize(1);
         }
 
         ref.get().finalizeGlobal(1);
-        Map<File, String> content = getFileContentByPath(outputFile);
-        assertEquals(1, content.size());
-        assertEquals("c=p1", content.keySet().iterator().next().getParentFile().getName());
-        assertEquals("a1,1\n" + "a2,2\n" + "a2,2\n" + "a3,3\n", content.values().iterator().next());
-        assertFalse(new File(tmpFile.toURI()).exists());
+        Map<File, String> content = getFileContentByPath(outputPath);
+        assertThat(content).hasSize(1);
+        assertThat(content.keySet().iterator().next().getParentFile().getName()).isEqualTo("c=p1");
+        assertThat(content.values().iterator().next())
+                .isEqualTo("a1,1\n" + "a2,2\n" + "a2,2\n" + "a3,3\n");
+        assertThat(new File(tmpPath.toUri())).doesNotExist();
     }
 
     @Test
-    public void testDynamicPartition() throws Exception {
+    void testDynamicPartition() throws Exception {
         AtomicReference<FileSystemOutputFormat<Row>> ref = new AtomicReference<>();
         try (OneInputStreamOperatorTestHarness<Row, Object> testHarness =
                 createSink(false, true, false, new LinkedHashMap<>(), ref)) {
             writeUnorderedRecords(testHarness);
-            assertEquals(2, getFileContentByPath(tmpFile).size());
+            assertThat(getFileContentByPath(tmpPath)).hasSize(2);
         }
 
         ref.get().finalizeGlobal(1);
-        Map<File, String> content = getFileContentByPath(outputFile);
+        Map<File, String> content = getFileContentByPath(outputPath);
         Map<String, String> sortedContent = new TreeMap<>();
         content.forEach((file, s) -> sortedContent.put(file.getParentFile().getName(), s));
 
-        assertEquals(2, sortedContent.size());
-        assertEquals("a1,1\n" + "a2,2\n" + "a3,3\n", sortedContent.get("c=p1"));
-        assertEquals("a2,2\n", sortedContent.get("c=p2"));
-        assertFalse(new File(tmpFile.toURI()).exists());
+        assertThat(sortedContent).hasSize(2);
+        assertThat(sortedContent.get("c=p1")).isEqualTo("a1,1\n" + "a2,2\n" + "a3,3\n");
+        assertThat(sortedContent.get("c=p2")).isEqualTo("a2,2\n");
```

Comment:
this _could_ be re-written to `contains(entry(...,...), ...)`

Reply:
done